shotover-proxy/tests/runner/hotreload_redis_tests.rs

```diff
@@ -0,0 +1,100 @@
+use crate::shotover_process;
+use redis::{Client, Commands};
+use std::time::Duration;
+use test_helpers::docker_compose::docker_compose;
+use tokio::time::sleep;
+
```

Comment:
These two tests should be moved into `hotreload_int_tests` and the existing test `test_hotreload_cli_flag_accepted` should be removed.
Since the new tests cover the same functionality and more, there is no point in maintaining it as a seperate test case.